src/Crowdin.Api/StringCorrections/AddCorrectionRequest.cs

```diff
@@ -0,0 +1,18 @@
+using System.ComponentModel;
+using JetBrains.Annotations;
+
+namespace Crowdin.Api.StringCorrections
+{
+    [PublicAPI]
+    public class AddCorrectionRequest
+    {
+        [Description("stringId")]
```

Comment:
We use `[JsonProperty]` attributes for `Newtonsoft.Json` serializer. Please replace `[Description]` attributes